### PR TITLE
Change style according to uiux meeting notes

### LINKED ISSUE #355: Implement style according to uiux minutes
The design team decided on the following:

https://docs.google.com/document/d/17xF1IFg9DijaoG7FqjECfsTLjVMV-SJhMlaT_77LV34/edit#heading=h.40n37oq105iv

We can implement this using theme/components and css.

### PR DESCRIPTION
**Pull request by navandstokes**
Closes #355 
Some points were not yet applicable. Will go through again in a while after other features are implemented.

### CODE REVIEW

File: app/frontend/src/components/Button.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ import React, { ElementType } from "react";
 import classNames from "classnames";
 
 const useStyles = makeStyles({
-  root: {},
+  root: {
+    borderRadius: 20,
```
Comment (lucaslcode) [inline app/frontend/src/components/Button.tsx]: Will it always be 20? If so you could do theme.shape.borderRadius and change it to 20 in the theme

Comment (lucaslcode) [reply app/frontend/src/components/Button.tsx]: Oh, I just saw the other line which is 10. What about `theme.shape.borderRadius*2`? Or maybe that's silly. In that case maybe we should remove from the theme file.

**Review by lucaslcode — DISMISSED** [review]

File: app/frontend/src/components/Button.tsx (hunk 1 above)
Comment (navandstokes) [reply app/frontend/src/components/Button.tsx]: Yeah I'm not sure. When trying to replicate some default material-ui components, if we use `theme.spacing()` and 8px grid, we have to use some fraction values. I think this is a discussion with the design team later on. Maybe it was too early to try to enforce defaults. 

Looking at the mid-fi again, it seems like cards radius is also 20. And just read about [Material Design guidelines](https://material.io/design/shape/applying-shape-to-ui.html#baseline-shape-values) for different border radius values for small/med/large components. So maybe `theme.shape.borderRadius*2` is a good idea.

File: app/frontend/src/features/search/SearchResult.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   card: {
-    borderRadius: theme.shape.borderRadius,
+    borderRadius: `${theme.shape.borderRadius * 2}px`,
```
Comment (navandstokes) [inline app/frontend/src/features/search/SearchResult.tsx:29]: Just `theme.shape.borderRadius * 2` was not working

**Review by lucaslcode — DISMISSED** [review]

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**